### ISSUE TITLE
Even though CMS side prompted error, successfully order placed on Arcadier end 

### ISSUE THREAD

**Issue by ThanChay** [Arcadier/Mogozay-Support#5]
If customer facing "billing address is required" error alert, it was successfully order placed on Arcadier end but when our admin side tried to click these invoice id, facing us "
Cannot access the requested page!" 

**Invoice ID**
you can search this two invoice id on Arcadier admin dashboard (transaction)
MOGOZAY157950984510BZ
MOGOZAY1579509914TM5U

![eb2b7e8a-a152-423f-8e49-ad7a2708f0cc](https://user-images.githubusercontent.com/6531873/72714653-7827cd00-3b9d-11ea-9f15-e9f48d89c715.jpeg)
![4da5e3b5-e26a-4d40-a5f5-289a17e533cc](https://user-images.githubusercontent.com/6531873/72714669-7e1dae00-3b9d-11ea-80ba-c8144f1a7b2f.jpeg)

![Screenshot from 2020-01-20 15-57-33](https://user-images.githubusercontent.com/6531873/72714748-a0173080-3b9d-11ea-8a43-370ae80a2fb7.png)

**Comment by Arcadier** [comment]
@ThanChay this is an issue stemming from the CMS, which is fixable also by the CMS. You can put the order on hold and contact your buyer to fill it up. Once Voilaah is done building their plug-in you should be able to manually fill up the billing address.
